<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/dddd.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bitstream Charter&quot;;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ts">
     <attribute name="title">
      <string>主厨推荐</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 11pt &quot;Helvetica&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>选择菜的编号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>显示所点的菜</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>特色菜</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 11pt &quot;Helvetica&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>450</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>450</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>选择菜的编号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>显示所点的菜</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>素菜</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>311</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 11pt &quot;Helvetica&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>选择菜的编号</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>440</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>540</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>显示所点的菜</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>肉菜</string>
     </attribute>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 11pt &quot;Helvetica&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>选择菜的编号</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_11">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>320</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>显示所点的菜</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>汤</string>
     </attribute>
     <widget class="QTableView" name="tableView_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 11pt &quot;Helvetica&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>选择菜的编号</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_14">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_15">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>320</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: oblique 13pt &quot;Helvetica&quot;;
color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>显示所选的菜</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
